static/app/views/issueDetails/groupReplays/replayClipPreviewWrapper.tsx

```diff
@@ -1,53 +0,0 @@
-import ReplayClipPreviewPlayer from 'sentry/components/events/eventReplay/replayClipPreviewPlayer';
-import {useReplayContext} from 'sentry/components/replays/replayContext';
-import type {Group} from 'sentry/types/group';
-import useLoadReplayReader from 'sentry/utils/replays/hooks/useLoadReplayReader';
-import type {ReplayColumn} from 'sentry/views/replays/replayTable/types';
-import type {ReplayListRecord} from 'sentry/views/replays/types';
-
-type Props = {
-  group: Group;
-  orgSlug: string;
-  pageLinks: string | null;
-  replaySlug: string;
-  replays: ReplayListRecord[] | undefined;
-  selectedReplayIndex: number;
-  setSelectedReplayIndex: (index: number) => void;
-  visibleColumns: ReplayColumn[];
-  overlayContent?: React.ReactNode;
-};
```

Comment:
Not all of these props were used. So they also came out of the parent components in `groupReplay.tsx`